src/utils/featheryClient/integrationClient.ts

```diff
@@ -345,12 +345,23 @@ export default class IntegrationClient {
     const { userId } = initInfo();
     const signer = fieldValues[action.envelope_signer_field_key];
     const payload: Record<string, any> = {
+      form_key: this.formKey,
       fuser_key: userId,
       documents: action.documents ?? [],
-      signer_email: signer
+      signer_email: signer,
+      use_quik: action.quik_documents
     };
-    if (action.quik_documents)
-      payload.quik = fieldValues[action.quik_json_field_key];
+
+    if (action.quik_tags_field_key) {
+      if (typeof fieldValues[action.quik_tags_field_key] === 'string') {
+        payload.quik_tags = (fieldValues[action.quik_tags_field_key] as string)
+          .split(',')
+          .map((tag) => tag.trim());
+      } else {
+        payload.quik_tags = fieldValues[action.quik_tags_field_key];
```

Comment:
this only works if the type is an array. we should also handle the case where the field value isn't an array or a string, in which case we should stringify it and wrap it with an array. also cleaner to do

const fieldVal = fieldValues[action.quik_tags_field_key];

and use that variable instead